avm/res/batch/batch-account/main.bicep

```diff
@@ -279,7 +279,7 @@ type diagnosticSettingType = {
   }[]?
 
   @description('Optional. A string indicating whether the export to Log Analytics should use the default destination type, i.e. AzureDiagnostics, or use a destination type.')
-  logAnalyticsDestinationType: ('Dedicated' | 'AzureDiagnostics' | null)?
+  logAnalyticsDestinationType: ('Dedicated' | 'AzureDiagnostics')?
```

Comment:
This will bump patch for all modules. I'd suggest to integrate this with the second PR after the one updating the CI to the new publishing naming convention

Reply:
Fair enough. But first we need to get that one through ;)